docs/persistent-volumes.md

```diff
@@ -0,0 +1,170 @@
+# Persistent Volumes and Claims
+
+This document describes the current state of Persistent Volumes in Kubernetes.  Familiarity with [volumes](./volumes.md) is suggested.
+
+A Persistent Volume (PV) is a piece of networked storage in the cluster that has been provisioned by an administrator.  It is a resource in the cluster just like a node is a cluster resource.   PVs are volume plugins like Volumes, but have a lifecycle independent of any individual pod that uses the PV.
```

Comment:
This API object captures the details of the implementation of the storage, be that NFS, iSCSI, or a cloud-provider-specific storage system.